pydoctor/model.py

```diff
@@ -77,7 +77,27 @@ class PrivacyClass(Enum):
     HIDDEN = 0
     PRIVATE = 1
     VISIBLE = 2
+    
+class KindClass(Enum):
```

Comment:
`KindClass` is more or less saying the same thing twice. Maybe name it just `Kind` or `DocumentableKind`?